resources/views/profile/index.blade.php

```diff
@@ -213,6 +213,23 @@ class="fa fa-coins mx-1"></i>{{$credits_reward_after_verify_discord}}
                                                             <p>You are verified!</p>
                                                         </div>
                                                     </div>
+                                                    <div class="row">
```

Comment:
this doesn't seem to have anything to do with better logs? for clarity make this a separate pull next time :)